pkg/redis/redis_test.go

```diff
@@ -537,7 +537,9 @@ func TestMutex(t *testing.T) {
 	case <-ctx.Done():
 		t.Fatalf("Timed out while waiting for LockMutex with a deadline to return")
 	case err := <-timeoutErrCh:
-		a.So(errors.IsDeadlineExceeded(err), should.BeTrue)
+		if !a.So(errors.IsDeadlineExceeded(err), should.BeTrue) {
```

Comment:
The error here actually was:
```
--- FAIL: TestMutex (1.03s)
    doc.go:66: 
        /home/adriansmares/go/src/go.thethings.network/lorawan-stack/pkg/redis/redis_test.go:540
        Expected: true
        Actual:   false
    redis_test.go:541: error:pkg/errors:net_operation (read tcp 127.0.0.1:55354->127.0.0.1:6379: i/o timeout)
```
Basically the commands couldn't be written in time, and `i/o timeout` is not a deadline exceeded error (error code wise).